src/libraries/StateDiffDecoder.sol

```diff
@@ -0,0 +1,536 @@
+                summary: "Required protocol version",
+                detail: "Unstructured storage slot for the required protocol version."
+            });
+        }
+        if (_slot == RECOMMENDED_SLOT) {
+            return DecodedSlot({
+                kind: "uint256",
+                oldValue: toUint(_oldValue),
+                newValue: toUint(_newValue),
+                summary: "Recommended protocol version",
+                detail: "Unstructured storage slot for the recommended protocol version."
+            });
+        }
+
+        // Gas paying token slots.
+        if (_slot == GAS_PAYING_TOKEN_SLOT) {
+            return DecodedSlot({
+                kind: "address",
+                oldValue: toAddress(_oldValue),
+                newValue: toAddress(_newValue),
+                summary: "Gas paying token address",
+                detail: "Unstructured storage slot for the address of the gas paying token."
+            });
+        }
+        if (_slot == GAS_PAYING_TOKEN_NAME_SLOT) {
+            return DecodedSlot({
+                kind: "string",
+                oldValue: vm.toString(_oldValue),
+                newValue: vm.toString(_newValue),
+                summary: "Gas paying token name",
+                detail: "Unstructured storage slot for the name of the gas paying token."
+            });
+        }
+        if (_slot == GAS_PAYING_TOKEN_SYMBOL_SLOT) {
+            return DecodedSlot({
+                kind: "string",
+                oldValue: vm.toString(_oldValue),
+                newValue: vm.toString(_newValue),
+                summary: "Gas paying token symbol",
+                detail: "Unstructured storage slot for the symbol of the gas paying token."
+            });
+        }
+    }
+
+    /// @notice Given a contract name and a slot, looks up the storage layout for the contract and
+    /// returns the decoded slot if it is found.
+    function tryStorageLayoutLookup(string memory _contractName, bytes32 _slot, bytes32 _oldValue, bytes32 _newValue)
+        internal
+        view
+        returns (DecodedSlot memory decoded_)
+    {
+        // Lookup the storage layout for the contract.
+        // TODO: For now this just uses the submodule's version of the monorepo. A future improvement
+        // would be to look up the latest release from the registry and fetch the storage layout
+        // from the monorepo at that tag.
+        string memory basePath = "/lib/optimism/packages/contracts-bedrock/snapshots/storageLayout/";
+        string memory artifactName = _contractName.endsWith("(GnosisSafe)") ? "GnosisSafe" : _contractName;
+        string memory path = string.concat(vm.projectRoot(), basePath, artifactName, ".json");
+
+        string memory storageLayout;
+        try vm.readFile(path) returns (string memory result) {
+            storageLayout = result;
+        } catch {
+            console.log("\x1B[33m[WARN]\x1B[0m Failed to read storage layout file at %s", path);
```

Comment:
I see three of these logs when I run: 

```
cd superchain-ops/tasks/sep/zora-002-fp-upgrade
SIMULATE_WITHOUT_LEDGER=1 just --dotenv-path $(pwd)/.env --justfile ../../../nested.just simulate foundation
```

They're being logged when we try to read the storage layout of the GnosisSafe. Is there any reason why we don't have the GnosisSafe storage layout at: `snapshots/storageLayout/`? Or how do we add it to the directory so this error doesn't happen?

Reply:
You may need to update your monorepo dep version with `forge install`, see https://github.com/ethereum-optimism/optimism/pull/14378